drivers/wifi/simplelink/Kconfig.simplelink

```diff
@@ -38,6 +39,37 @@ config WIFI_SIMPLELINK_MAX_SCAN_RETRIES
 	  The number of times, separated by a one second interval, to retry
 	  a request for the network list.
 
+config WIFI_SIMPLELINK_WLAN_POLICY_AUTO_CONNECT
+	bool "Use the SimpleLink Auto Connection Policy"
+	default y
+	help
+	  The device tries to connect to an AP from the stored profiles
+	  based on priority. Up to seven profiles are supported.
+
+config WIFI_SIMPLELINK_WLAN_POLICY_FAST_CONNECT
+	bool "Use the SimpleLink Fast Connection Policy"
+	default y
+	help
+	  The device will try to connect to the last connected AP.
+
+config WIFI_SIMPLELINK_WLAN_POLICY_ANYP2P
+	bool "Use the SimpleLink AnyP2P Connection Policy"
+	default n
```

Comment:
That is not needed, please remove it. Please see https://docs.zephyrproject.org/1.14.0/guides/kconfig/index.html#redundant-defaults